test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5185,6 +5185,9 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
 
         if delnodes:
             self.delete_nodes()
+        # disable cgroups hook - mom revert will enable only if needed
```

Comment:
I'm a bit confused, doesn't the "if reverthooks:" section below this take care of the cgroups hook? If not, then I think we should fix that instead of especially disabling the cgroups hook here. The "if reverthooks:" section is supposed to disable all hooks